code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Since these are being completely replaced, delete them. We'll revert if we decide to go back to non-modular AKs.